easy_study/lib/model/Subject.dart

```diff
@@ -75,11 +82,6 @@ class Subject {
 
   @override
   String toString() {
-    return 'Subject{_title: $_title, _room: $_room,'
-        ' _description: $_description, _id: $_id, _timeSpent:'
-        ' $_timeSpent, _hoursWeek: $_hoursWeek, _type: $_type, '
-        '_priority: $_priority, _color: $_color, _dueDate: $_dueDate}';
+    return 'Subject{_title: $_title, _room: $_room, _description: $_description, _id: $_id, _timeSpent: $_timeSpent, _hoursWeek: $_hoursWeek, _type: $_type, _priority: $_priority, _startTimetracking: $_startTimetracking, _color: $_color, _dueDate: $_dueDate}';
```

Comment:
Don't go over the length of a single line which is 100.
When u format the code, maybe select the code to format before formatting the whole file. If the auto format doesn't work, do it manually.